spec/handlers/newflow/student_signup_spec.rb

```diff
@@ -7,7 +7,7 @@ module Newflow
         load('db/seeds.rb')
       end
 
-      let(:result) do
+      let(:subject) do
```

Comment:
consider a named subject, to add context.  what is this?   :student_signup? a special one? what kind of one?

Reply:
Considered. Simply `subject` is perfectly fine IMO in this case.